### ISSUE TITLE
Templates under ietf/secr need

### ISSUE THREAD

**Issue by larseggert** [ietf-tools/datatracker#3785]
### What happened?

The templates under `ietf/secr` for historic reasons use different base templates than the rest of the datatracker. Those base templates and most individual templates under `ietf/secr` are not written in HTML5 and should be updated.

My personal preference would be to use the overall base templates here, too.

### What browser(s) are you seeing the problem on?

Not Applicable

### Code of Conduct

- [X] I agree to follow the IETF's Code of Conduct

**Comment by rjsparks** [comment]
Many (most) of the functionality from the  /secr views and templates have already been moved into the core apps. (/secr/sreq is a big outlier). A project to remove any that are no longer needed should complete before spending effort fixing the html in the ones that are left.
